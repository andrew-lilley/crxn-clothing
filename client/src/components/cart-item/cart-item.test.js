import React from 'react';
import { shallow } from 'enzyme';
import CartItem from './cart-item.component';

test('should render CartItem component', () => {
  const mockItem = {
    imageUrl: 'images/shop-img/sneakers/adidas-nmd.png',
    price: 220,
    name: 'Adidas NMD',
    quantity: 2
  };

  expect(shallow(<CartItem item={mockItem} />)).toMatchSnapshot();
});