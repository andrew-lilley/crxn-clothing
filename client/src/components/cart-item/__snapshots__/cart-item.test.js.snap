// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render CartItem component 1`] = `
<styled.div>
  <styled.img
    alt="Adidas NMD"
    src="adidas-nmd.png"
  />
  <styled.div>
    <span>
      Adidas NMD
    </span>
    <span>
      2
       x 
      <Connect(Price)
        price={220}
      />
    </span>
  </styled.div>
</styled.div>
`;
